### PR TITLE
[Improve] Rename k8s to Kubernetes

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/flink-k8s/1-deployment.md [hunk 1]
```diff
@@ -81,10 +81,10 @@ docker pull <your_register_addr>/streampark/busybox
...
 * **Flink Base Docker Image**： 基础 Flink Docker 镜像的 Tag，可以直接从 [DockerHub - offical/flink](https://hub.docker.com/_/flink) 获取，也支持用户私有的底层镜像，此时在 setting 设置 Docker Register Account 需要具备该私有镜像 	`pull` 权限。
 * **Rest-Service Exposed Type**：对应 Flink 原生 [kubernetes.rest-service.exposed.type](https://ci.apache.org/projects/flink/flink-docs-stable/docs/deployment/config/#kubernetes) 配置，各个候选值说明：
-  * `ClusterIP`：需要 StreamPark 可直接访问 K8s 内部网络；
-  * `LoadBalancer`：需要 K8s 提前创建 LoadBalancer 资源，且 Flink Namespace 具备自动绑定权限，同时 StreamPark 可以访问该 LoadBalancer 网关；
-  * `NodePort`：需要 StreamPark 可以直接连通所有 K8s 节点；
-* **Kubernetes Pod Template**： Flink 自定义 pod-template 配置，注意container-name必须为flink-main-container，如果k8s pod拉取docker镜像需要秘钥，请在pod template文件中补全秘钥相关信息，pod-template模板如下：
+  * `ClusterIP`：需要 StreamPark 可直接访问 Kubernetes 内部网络；
+  * `LoadBalancer`：需要 Kubernetes 提前创建 LoadBalancer 资源，且 Flink Namespace 具备自动绑定权限，同时 StreamPark 可以访问该 LoadBalancer 网关；
+  * `NodePort`：需要 StreamPark 可以直接连通所有 Kubernetes 节点；
+* **Kubernetes Pod Template**： Flink 自定义 pod-template 配置，注意container-name必须为flink-main-container，如果Kubernetes pod拉取docker镜像需要秘钥，请在pod template文件中补全秘钥相关信息，pod-template模板如下：
```
Comment (tisonkun) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/flink-k8s/1-deployment.md:87]:
```suggestion
* **Kubernetes Pod Template**： Flink 自定义 pod-template 配置，注意 `container-name` 必须为 `flink-main-container`，如果 Kubernetes pod 拉取 Docker 镜像需要秘钥，请在 pod 模板文件中补全秘钥相关信息，pod 模板如下：
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/flink-k8s/1-deployment.md [hunk 2]
```diff
@@ -98,21 +98,21 @@ docker pull <your_register_addr>/streampark/busybox
       imagePullSecrets:
       - name: regsecret
     ```
-* **Dynamic Option**： Flink on k8s动态参数（部分参数也可在pod-template文件中定义），该参数需要以-D开头，详情见[Flink on Kubernetes相关参数](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes)
+* **Dynamic Option**： Flink on Kubernetes动态参数（部分参数也可在pod-template文件中定义），该参数需要以-D开头，详情见[Flink on Kubernetes相关参数](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes)
```
Comment (tisonkun) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/flink-k8s/1-deployment.md:101]:
```suggestion
* **Dynamic Option**： Flink on Kubernetes 动态参数（部分参数也可在 pod-template 文件中定义），该参数需要以 `-D` 开头，详情见 [Flink on Kubernetes相关参数](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes)。
```

File: docs/flink-k8s/1-deployment.md [hunk 3]
```diff
@@ -80,11 +80,11 @@ parameter descriptions are as below：
...
+* **Rest-Service Exposed Type**：Description of candidate values for native Flink Kubernetes configuration [kubernetes.rest-service.exposed.type](https://ci.apache.org/projects/flink/flink-docs-stable/docs/deployment/config/#kubernetes) ：
   * `ClusterIP`：ip that StreamPark can access；
   * `LoadBalancer`：resource of LoadBalancer should be allocated in advance， Flink Namespace own permission of automatic binding，and StreamPark can access LoadBalancer`s gateway；
-  * `NodePort`：StreamPark can access  all K8s nodes；
-* **Kubernetes Pod Template**： It`s Flink custom configuration of pod-template.The container-name must be flink-main-container. If the k8s pod needs a secret key to pull the docker image, please fill in the information about
+  * `NodePort`：StreamPark can access  all Kubernetes nodes；
+* **Kubernetes Pod Template**： It`s Flink custom configuration of pod-template.The container-name must be flink-main-container. If the Kubernetes pod needs a secret key to pull the docker image, please fill in the information about
```
Comment (tisonkun) [inline docs/flink-k8s/1-deployment.md:87]:
```suggestion
* **Kubernetes Pod Template**: This is Flink's custom configuration of pod-template. The `container-name` must be `flink-main-container`. If the Kubernetes pod needs a secret key to pull the Docker image, please fill in the information about
```

File: docs/flink-k8s/1-deployment.md [hunk 4]
```diff
@@ -80,11 +80,11 @@ parameter descriptions are as below：
...
 the secret key in the pod template file.The example pod-template is as below：
```
Comment (tisonkun) [inline docs/flink-k8s/1-deployment.md:88]:
```suggestion
the secret key in the pod template file. Below is an example of pod-template:
```

File: docs/flink-k8s/1-deployment.md [hunk 5]
```diff
@@ -101,7 +101,7 @@ the secret key in the pod template file.The example pod-template is as below：
       - name: regsecret
     ```
 
-* **Dynamic Option**：It`s dynamic parameter of Flink on k8s（part of parameters can also be defined in pod-template）which should start with -D.Details are in[Flink on Kubernetes parameters](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes)
+* **Dynamic Option**：It`s dynamic parameter of Flink on Kubernetes（part of parameters can also be defined in pod-template）which should start with -D.Details are in[Flink on Kubernetes parameters](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes)
```
Comment (tisonkun) [inline docs/flink-k8s/1-deployment.md:104]:
```suggestion
* **Dynamic Option**: Dynamic options of Flink on Kubernetes (part of parameters can also be defined in pod-template). Options should start with `-D`. Details can be found at [Flink on Kubernetes parameters](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes).
```

File: docs/flink-k8s/1-deployment.md [hunk 6]
```diff
@@ -114,7 +114,7 @@ The additional configuration of Flink-Native-Kubernetes Session Job will be deci
 
 ## other configuration
 
-StreamPark parameter related to Flink-K8s in `applicaton.yml` are as below.And in most condition, it is no need to change it.
+StreamPark parameter related to Flink-Kubernetes in `applicaton.yml` are as below.And in most condition, it is no need to change it.
```
Comment (tisonkun) [inline docs/flink-k8s/1-deployment.md:117]:
```suggestion
StreamPark parameters related to Flink-Kubernetes in `applicaton.yml` are as below. And in most conditions, there is no need to change it.
```

**Comment by caicancai** [comment]
@tisonkun thank you

**Comment by tisonkun** [comment]
Please be careful to type. In your original commit title and pr title, you wrote `reanme` and this is why we have so many text to correct in the document ...

**Comment by tisonkun** [comment]
You may use tools like Grammarly for help, and perhaps [typos](https://github.com/crate-ci/typos) to check files.

**Comment by caicancai** [comment]
Thanks for the reminder, I will pay attention to it in the future

**Pull request merged**